pkg/fanal/analyzer/language/nodejs/license.go

```diff
@@ -0,0 +1,110 @@
+package nodejs
+
+import (
+	"errors"
+	"io/fs"
+	"path"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/nodejs/packagejson"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/licensing"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/utils/fsutils"
+)
+
+func ParseLicenses(
```

Comment:
`npm` also finds licenses in `node_modules/<package-name>/package.json` file.
Maybe we can use this function? or will these changes be in another PR?

npm also finds licenses in the node_modules/<package-name>/package.json file.
Maybe we can use this feature? Or will these changes be in another PR?

Reply:
How about making these changes in another PR?

Reply:
Okay. I created #4964 for this task.